WordPress/src/test/java/org/wordpress/android/sharedlogin/resolver/SharedLoginResolverTest.kt

```diff
@@ -204,13 +211,18 @@ class SharedLoginResolverTest {
         featureEnabled()
         classToTest.tryJetpackLogin()
         verify(sharedLoginAnalyticsTracker, never()).trackLoginSuccess()
-        verify(sharedLoginAnalyticsTracker, times(1)).trackLoginFailed(ErrorType.QueryTokenError)
+        verify(sharedLoginAnalyticsTracker, times(1)).trackLoginFailed(ErrorType.QueryLoginDataError)
     }
 
     @Test
-    fun `Should track login failed if access token IS empty`() {
+    fun `Should track login failed if access token IS empty and no self-hosted sites`() {
```

Comment:
Would it make sense to have two unit tests instead of just this one?

This is our `if` expression:
```
if (accessToken.isNotEmpty() || selfHostedSites.isNotEmpty())
```

AFAIU this test is checking `if (false || false)`. I was thinking that maybe we could have a test for `if (false || true)`, one for `if (true || false)` and another for `if (true || true)`.

Reply:
Yep, I was fixing the existing unit test but planned to add some cases (now added) after resolving the CI 👍